internal/updater/executor.go

```diff
@@ -0,0 +1,158 @@
+package updater
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"hash/fnv"
+	"io"
+	"sync"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/rs/zerolog"
+
+	"github.com/quay/claircore/internal/vulnstore"
+	"github.com/quay/claircore/internal/vulnstore/postgres"
+	"github.com/quay/claircore/libvuln/driver"
+)
+
+const tryLock = `SELECT pg_try_advisory_xact_lock($1);`
+
+func crushkey(key string) int64 {
```

Comment:
can we please not do this? It is not helpful when troubleshooting to look at pg_advisory locks and have to decode to know if something is keeping a lock around needlessly.